test/e2e/components/vl-cookie-consent.js

```diff
@@ -14,17 +14,19 @@ class VlCookieConsent extends VlElement {
     }
 
     async getOptIn(label) {
-        const modal = await this._getModal();
-        const element = await modal.findElement(By.css('vl-cookie-consent-opt-in[data-vl-label="' + label + '"]'));
-        return new VlCookieConsentOptIn(this.driver, element);
+        const optIns = await this.getOptIns();
+        for (let i = 0; i < optIns.length; i++) {
+            const optIn = optIns[i];
+            if (await optIn.getLabel() === label) {
+                return optIn;
+            }
```

Comment:
array.find geen optie hier?

Reply:
Nee, vanwege de async-await die we moeten doen.